I need to fix travis:
```
build/node_modules/bower/bin/bower install

make: build/node_modules/bower/bin/bower: Command not found

make: *** [core/vendor] Error 127
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -8,7 +8,7 @@ cache:
     - $HOME/.cache/bower
     - $HOME/.composer/cache
     - $HOME/.npm
-    - build/node_modules
+    - /tmp/phantomjs
 
 env:
   global:

--- a/Makefile
+++ b/Makefile
@@ -110,11 +110,7 @@ clean-composer-deps:
 #
 $(nodejs_deps): build/package.json
 	$(NPM) install --prefix $(NODE_PREFIX)
-
-# in some cases installing bower alone could be enough
-$(BOWER):
-	$(NPM) install --prefix $(NODE_PREFIX) bower
-	touch $(BOWER)
+	touch $(nodejs_deps)
 
 .PHONY: install-nodejs-deps
 install-nodejs-deps: $(nodejs_deps)
@@ -125,7 +121,7 @@ clean-nodejs-deps:
 
 #
 # ownCloud core JS dependencies
-$(core_vendor): $(BOWER) bower.json
+$(core_vendor): $(nodejs_deps) bower.json
 	$(BOWER) install
 
 .PHONY: install-js-deps

